srtcore/core.cpp

```diff
@@ -7855,7 +7855,17 @@ int CUDT::processData(CUnit* unit)
 
    }
    if ( !lossdata.empty() )
-       sendCtrl(UMSG_LOSSREPORT, NULL, lossdata.data(), lossdata.size());
+   {
+       sendCtrl(UMSG_LOSSREPORT, NULL,
+#if (__cplusplus >= 201103L)
+           // ::std::vector<>.data() is c++11.
+           lossdata.data(),
```

Comment:
Is there any value in keeping the C++11 version around if the "portable" variant works everywhere?

Reply:
I dont think so. I will remove the c++11 version and update the PR.

Reply:
Submitted update to the PR that removes the c++11 version in favor of the portable one.